_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,42 @@
+categories: [ audio ]
+image: "mixer.webp"
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
+How much does it cost?
+
+### - Introduction
+
+Audio mixing, what is it? Audio Mixing is where you take multiple input sources of audio and put them into a single or fewer number output signals. An example of input sources would be a microphone, game audio, stream alert audio. An example of an output signal would be our headphones, speakers, or stream. Can you achieve audio mixing for streams and videos without a digital or physical audio mixer? Mostly, but in practice it is significantly more complicated to route audio properly without using some form of audio mixing software or hardware when dealing with multiple sources.
+
+### - Goals of Audio Mixing
+
+The goals of audio mixing can be as simple as taking a microphone and game audio then pushing it to a single output. They can also be as elaborate as dividing out a microphone, or even two or three microphones, game audio, alerts, computer sounds, and music. Each having their own source to pull audio from then pushing that audio to two or more outputs, headphones, broadcast software such as OBS or SLOBS, or discord. Of course we don't want our stream audio going to friends on discord, and we don't want our mic audio going to our speakers if we aren't using headphones at the moment.
+
+This is where audio mixing software and hardware comes into play. Audio mixing allows us to properly route our audio to where it needs to be and often times gives it a boost in quality, while also making it easier to handle unlike windows sound settings and such. Having more options can be daunting at times, especially when just starting out, but once you get the hang of it, more often than not it saves so much more time and provides you with better quality than if you just went without.
+
+### - Audio Mixing options
+
+Are you interested in having more control over your audio? Lets go over our options then!
+
+###### - Mixing in OBS?
+You might say OBS has its own audio mix, that is true. However, just like using windows, it can be complicated. First you have to find audio levels in windows that suite yourself, so you can hear the game and other things. Then you have to pull the audio directly into OBS and try and manage it there. It can be a complete mess because the audio levels are already being adjusted for your hearing first. Optionally, if you adjust for the OBS Mix first, the levels for you hearing the audio will also be affected, too high, too low, and when you adjust the audio levels going to your headphones it will then adjust the levels in OBS. Big wonky mess right?
+
+###### - GoXLR, GoXLR-Mini, Elgato Wave
+ I'm sure every streamer has heard of a GoXLR, or GoXLR-Mini which come with software mixing alongside hardware controls. There is also the Elgato Wave microphones which come with similar software mixing and some hardware controls.
```

Comment:
Drop the comma after GoXLR AND/OR surround "or GoXLR-Mini" with parenthesis.

Reply:
May want to add a blurb about not going to in depth on this one as we don't have anyone on the team that has one of these microphones, and while they sound very cool, we can't comment much on them due to the lack of owning one.

Reply:
Added: Between the current writers for the Library by Stream Sounds, no one has either a GoXLR, GoXLR-Mini, or Elgato Wave microphone. We are interested in pulling in someone who does have these devices and would like to add some articles on these devices to the Library.